third_party/pcsc-lite/naclport/server_clients_management/src/client-handler-unittest.js

```diff
@@ -17,7 +17,9 @@
 
 goog.require('GoogleSmartCard.PcscLiteServerClientsManagement.ClientHandler');
 goog.require('goog.Promise');
+goog.require('goog.reflect');
 goog.require('goog.testing');
+goog.require('goog.testing.asserts');
```

Comment:
nit: Swap this with the next line, to keep directives sorted.

Reply:
oops, fixed